class.jetpack-gutenberg.php

```diff
@@ -326,6 +326,20 @@ public static function get_jetpack_gutenberg_extensions_whitelist() {
 		return $preset_extensions;
 	}
 
+	/**
+	 * Returns a diff from a combined list of whitelisted extensions and extensions determined to be excluded
+	 *
+	 * @param  array $whitelisted_extensions An array of whitelisted extensions.
+	 *
+	 * @return array A list of blocks: eg [ 'publicize', 'markdown' ]
+	 */
+	public static function get_available_extensions( $whitelisted_extensions = null ) {
```

Comment:
Is the `$whitelisted_extensions` argument still necessary? It doesn't seem to be used anywhere else other than in tests.